### PR TITLE
[test](auth) Add call auth module case

### PR DESCRIPTION
**Pull request by zfr9527**

### CODE REVIEW

**Comment by zfr9527** [comment]
run buildall

**Comment by zfr9527** [comment]
run buildall

**Comment by zfr9527** [comment]
run buildall

**Comment by zfr9527** [comment]
run buildall

**Comment by zfr9527** [comment]
run buildall

File: regression-test/data/auth_call/multi_load_cm.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (morrySnow) [inline regression-test/data/auth_call/multi_load_cm.sh:2]: what's this?

**Review by morrySnow — DISMISSED** [review]

File: regression-test/data/auth_call/multi_load_cm.sh (hunk 1 above)
Comment (zfr9527) [reply regression-test/data/auth_call/multi_load_cm.sh:2]: The case will write data into this file and then execute. So this file is temporarily empty.

**Comment by zfr9527** [comment]
run buildall

**Comment by zfr9527** [comment]
run buildall

File: regression-test/data/auth_call/multi_load_cm.sh (hunk 1 above)
Comment (zfr9527) [reply regression-test/data/auth_call/multi_load_cm.sh]: The empty file has been deleted and generated every time it is executed.

**Review by morrySnow — APPROVED** [review]

**Review by zddr — APPROVED** [review]

**Pull request merged**